plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -2812,12 +2821,13 @@ public function delete_meta( &$object, $meta ) {
 	 * @return int|bool  meta ID or false on failure
 	 */
 	public function add_meta( &$object, $meta ) {
-		$add_meta = $this->data_store_meta->add_meta( $object, $meta );
-		$this->after_meta_change( $object, $meta );
+		$add_meta        = $this->data_store_meta->add_meta( $object, $meta );
+		$meta->id        = $add_meta;
+		$changes_applied = $this->after_meta_change( $object, $meta );
```

Comment:
I'm not sure it's essential, but should we do anything if this returns false? Atm I don't believe we do anything with `$changes_applied`.

Reply:
Sorry, deleting ... I meant to apply this comment to line 2804.